src/main/java/book/store/service/impl/BookServiceImpl.java

```diff
@@ -11,14 +16,26 @@
 @Service
 public class BookServiceImpl implements BookService {
     private final BookRepository bookRepository;
+    private final BookRequestDtoMapper bookRequestDtoMapper;
+    private final BookResponseDtoMapper bookResponseDtoMapper;
 
     @Override
-    public Book save(Book book) {
-        return bookRepository.save(book);
+    public BookDto createBook(CreateBookRequestDto requestDto) {
+        return bookResponseDtoMapper.toDto(bookRepository
+                .save(bookRequestDtoMapper.toModel(requestDto)));
     }
 
     @Override
-    public List<Book> findAll() {
-        return bookRepository.findAll();
+    public BookDto getBookById(Long id) {
+        Book book = bookRepository.findById(id)
+                .orElseThrow(() -> new EntityNotFoundException("Don't find book with id " + id));
+        return bookResponseDtoMapper.toDto(book);
```

Comment:
inline expression using Optional#map method
```suggestion
        Book book = bookRepository.findById(id)
                .orElseThrow(() -> new EntityNotFoundException("Don't find book with id " + id));
        return bookResponseDtoMapper.toDto(book);
```